tests/e2e/util.go

```diff
@@ -3339,47 +3338,54 @@ func toggleCSIMigrationFeatureGatesOnKubeControllerManager(ctx context.Context,
 		sshCmd = "sed -i '/CSIMigration/d' " + kcmManifest
 	}
 	grepCmd := "grep CSIMigration " + kcmManifest
-	k8sMasterIP := getK8sMasterIP(ctx, client)
-	framework.Logf("Invoking command '%v' on host %v", grepCmd, k8sMasterIP)
-	sshClientConfig := &ssh.ClientConfig{
-		User: "root",
-		Auth: []ssh.AuthMethod{
-			ssh.Password("ca$hc0w"),
-		},
-		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
-	}
-	result, err := sshExec(sshClientConfig, k8sMasterIP, grepCmd)
-	if err != nil {
-		return err
-	}
-	if err != nil {
-		fssh.LogResult(result)
-		return fmt.Errorf("command failed/couldn't execute command: %s on host: %v , error: %s",
-			grepCmd, k8sMasterIP, err)
-	}
-	if result.Code != 0 {
-		if add {
-			// nolint:misspell
-			sshCmd = "gawk -i inplace '/--bind-addres/ " +
-				"{ print; print \"    - --feature-gates=CSIMigration=true,CSIMigrationvSphere=true\"; next }1' " +
-				kcmManifest
-		} else {
-			return nil
+	k8sMasterIPs := getK8sMasterIPs(ctx, client)
+	for _, k8sMasterIP := range k8sMasterIPs {
+		framework.Logf("Invoking command '%v' on host %v", grepCmd, k8sMasterIP)
+		sshClientConfig := &ssh.ClientConfig{
+			User: "root",
+			Auth: []ssh.AuthMethod{
+				ssh.Password(esxPassword),
```

Comment:
Any reason why we are using esxPassword for k8sMasterVm ?

Reply:
Changed to use a different variable